io.github.atari800.Atari800.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.github.atari800.Atari800
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: atari800
+build-options:
+  no-debuginfo: true
+  strip: true
+modules:
+  - name: libsdl
+    sources:
+      - type: archive
+        url: https://libsdl.org/release/SDL-1.2.15.tar.gz
+        sha256: d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00
+      - type: patch
+        path: patch/sdlpatch.txt
+        strip-components: 1
+  - name: atari800
+    config-opts:
+      - --with-video=sdl
+      - --with-sound=sdl
+      - --enable-pngcodec
+      - --with-mp3=lame
+    post-install:
+      - install -Dm 0644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm 0644 atari800-16x16.png ${FLATPAK_DEST}/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm 0644 atari800-24x24.png ${FLATPAK_DEST}/share/icons/hicolor/24x24/apps/${FLATPAK_ID}.png
+      - install -Dm 0644 atari800-32x32.png ${FLATPAK_DEST}/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm 0644 atari800-48x48.png ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm 0644 atari800-64x64.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
+      - install -Dm 0644 atari800-128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm 0644 atari800-256x256.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - rm ${FLATPAK_DEST}/bin/sdl-config
+      - rm -rf ${FLATPAK_DEST}/include
+      - rm -rf ${FLATPAK_DEST}/lib/pkgconfig
+      - rm ${FLATPAK_DEST}/lib/*.a
+      - rm ${FLATPAK_DEST}/lib/*.la
+    sources:
+      - type: git
+        url: https://github.com/atari800/atari800.git
+        tag: ATARI800_5_0_0
+        commit: c0f94a7bf15543d8feb87ba046c456087c9d8227
+      - type: dir
+        path: misc
+      - type: dir
+        path: icons
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=home
```

Comment:
I need to create and rw ~/.atari800.cfg.  I originally had:
--filesystem=~/.atari800.cfg
--filesystem=xdg-documents

However, this only worked if ~/.atari800.cfg already existed.  If it was not there, it did not get created.  I messed around with --persist which looked like it might be the solution, but could not get that to work.  So, I just used --filesystem=home instead.

How do I allow access to the one file in ~/ and also allow it to be created?

Reply:
This depends on the app. Possibly `--persist=.` would do what you want, but it needs additional tests to see how it interfacts with opening files.

Reply:
"--persist=." worked.  Updated yml.  Thanks.